changelog.sh

```diff
@@ -70,6 +70,11 @@ if [ ! -z "$INPUT_FUTURE_RELEASE" ]; then
   ARG_FUTURE_RELEASE="--future-release $INPUT_FUTURE_RELEASE"
 fi
 
+ARG_ISSUE_LINE_LABELS=""
+if [ ! -z "$INPUT_ISSUE_LINE_LABELS" ]; then
+  ARG_ISSUE_LINE_LABELS="--issue-line-labels $INPUT_ISSUE_LINE_LABELS"
+fi
+
 # build changelog
 echo "Running: github_changelog_generator $ARG_USER $ARG_PROJECT $ARG_RELEASE_BRANCH $ARG_BASE $ARG_OUTPUT $ARG_TOKEN $ARG_SINCE_TAG $ARG_DUE_TAG $ARG_EXCLUDE_TAGS $ARG_EXCLUDE_LABELS $ARG_DATE_FORMAT $ARG_FUTURE_RELEASE"
```

Comment:
Everything looks good, if you just add the arg to this echo as well for easier debugging, I'll merge it to master.

Reply:
Thanks, fixed!